DuckDB assets (#19347)
DataDog/integrations-core: duckdb/README.md

## Overview

DuckDB is a high-performance analytical database system. It is available as a standalone CLI application and has clients for Python, R, Java, Wasm, etc., with deep integrations with packages such as pandas and dplyr.

> For more information on using DuckDB, please refer to the [DuckDB documentation][9].

Reviewer: ```suggestion
For more information on using DuckDB, refer to the [DuckDB documentation][9].
```

This check monitors [DuckDB][1] through the Datadog Agent. 

## Setup

DuckDB has two configurable options for concurrency:

- One process can both read and write to the database.
- Multiple processes can read from the database, but no processes can write (access_mode = 'READ_ONLY').

<div class="alert alert-warning">

> The datadog agent is using the read_only mode to get metrics, with a default 60 seconds frequency (min_collection_interval). 

Reviewer: ```suggestion
The Datadog Agent uses the <code>read_only</code> mode to get metrics, with a default frequency of 60 seconds (<code>min_collection_interval</code>). 
```

> Feel free to increase this value to reduce concurrency issues.

Reviewer: ```suggestion
You can increase this value to reduce concurrency issues.
```
